draft-ietf-tls-tls13.md

```diff
@@ -1615,7 +1620,7 @@ Structure of this message:
            ProtocolVersion version;
            Random random;
            CipherSuite cipher_suite;
-           Extension extensions<0..2^16-1>;
+           Extension extensions<1..2^16-1>;
```

Comment:
I did not go through and calculate the actual minimum, but I'm pretty sure you need to send at least one extension in the ServerHello.